### DISCUSSION TITLE
Adding a variable request in the menu type

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#881] [Q&A]
hi 
I want to add some request variables in the menu type but I do not know how to do this in JCB.
Is this possible at all in JCB?

**Comment by Llewellynvdm** [comment]
Wow I am not sure I understand... if your asking what I think your asking then the answer is yes.

We do that in the Sermon Distributor component... sorry I realize it not the ideal demo component for you to use... but it is where I demonstrate this.

You basically use a global configuration field in a tab with the same name as the site view your targeting. With a few more tweaks in the field... let me see, I would think there is a tutorial about this.

- [Manage a Components Global Config Option Field in Relation with Menu Params](https://www.youtube.com/watch?v=dhVVQP4KS3Q&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=83)
- [Setup Menu Links, and Item Links, even Category Links](https://www.youtube.com/watch?v=tgtonbyyjDs&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=91)

These should probably point you in the right direction.

**Comment by GiuseppeBosco** [comment]
Hello everyone,

I'm encountering an issue with the configuration of a JCB component and I'm hoping for some assistance. I need to add a filtering field to a menu item, but I'm struggling to find the correct way to implement it.

I've already tried following the video tutorials and discussions on this forum, but I haven't been able to find a solution that applies to my specific case. It seems the instructions often refer to older JCB versions or describe more complex functionalities than what I require.

My current setup is straightforward:

I have a list view (cani) that displays a list of dogs of various breeds, presented as cards.

Each record includes a razza (breed) field.

For the razza field, which I use to set the breed when inserting a single record, I've implemented a custom field that pulls the breed from a specific database table. I've also attempted to set its display property to menu, but this change hasn't resulted in any visible field within the menu item configuration.

My goal is to add this single field to the menu item configuration linked to this view. This would allow users to select a specific "tipologia" (breed) and filter the displayed records accordingly. I'm not looking for a complex system with multiple fields, just this straightforward filtering option.

Could anyone guide me on the correct method to insert a field into the menu item settings to filter the view's results?

Thank you very much for your help.
